app/models/katello/product.rb

```diff
@@ -149,6 +149,10 @@ def anonymous?
       provider.anonymous_provider?
     end
 
+    def used_by_other_org
+      ::Katello::Product.where(["cp_id = ? AND id != ?", cp_id, id]).count > 0
+    end
+
```

Comment:
Should this end in a `?` since it is a boolean result? Should that be 'another' instead of 'other' ?  You could do `self.class.where` to make this robust against module or class name changes I think.